lib/zone-spec/fake-async-test.ts

```diff
@@ -339,7 +339,9 @@
                   this._setInterval(task.invoke, task.data['delay'], (task.data as any)['args']);
               break;
             case 'XMLHttpRequest.send':
-              throw new Error('Cannot make XHRs from within a fake async test.');
+              throw new Error(
+                  'Cannot make XHRs from within a fake async test. Request URL: ' +
+                  (task.data as any)['url']);
```

Comment:
@vikerman , if we get the url from `task.data`, maybe we can get the url by directly calling 'task.data.target.responseURL`?

Reply:
From my testing responseURL is an empty string before the actual request is sent.

Reply:
@vikerman , yes, you are right!